assets/prefabs/food/raw/Coconut.prefab

```diff
@@ -7,10 +7,13 @@
     "DisplayName": {
         "name": "Coconut"
     },
+    "CraftingStationIngredient": {
+            "type": "Cooking:coconut"
+    }
```

Comment:
Missing comma here. It should be `},`

If there's another component following a definition (like here), there should be a comma following the end brace.